### PR TITLE
Add RDM.app v2.5.0

### PR DESCRIPTION
**Pull request by MiroDojkic**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew install --cask {{cask_file}}` worked successfully.
- [x] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Why submit a fork instead of the original?

**Comment by MiroDojkic** [comment]
@vitorgalvao fork supports Big Sur, has additional features and it received a bunch of updates, while the original seems abandoned ever since 5 years ago. 😅

**Comment by vitorgalvao** [comment]
But there’s indication this one is the fork to use. [This needs a name change](https://github.com/Homebrew/brew/blob/master/docs/Acceptable-Casks.md#forks-and-apps-with-conflicting-names).

**Comment by MiroDojkic** [comment]
@vitorgalvao the [original](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/avibrazil-rdm.rb) is also prefixed with the vendor because of the name clash. What do you suggest I rename it to?

File: Casks/usr-sse2-rdm.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+cask "usr-sse2-rdm" do
+  version "2.5.0"
+  sha256 "9d583e43742018e45bad79c8af9e0accec6b6a10d94c60ac7d31721bbb3c237e"
+
+  url "https://github.com/usr-sse2/RDM/releases/download/#{version}/RDM.zip"
+  name "RDM"
+  desc "Set a Retina display to custom resolutions"
+  homepage "https://github.com/usr-sse2/RDM"
+
+  livecheck do
```
Comment (vitorgalvao) [inline Casks/usr-sse2-rdm.rb:10]: This shouldn’t be necessary. Try it without.

**Review by vitorgalvao — DISMISSED** [review]

File: Casks/usr-sse2-rdm.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+cask "usr-sse2-rdm" do
+  version "2.5.0"
+  sha256 "9d583e43742018e45bad79c8af9e0accec6b6a10d94c60ac7d31721bbb3c237e"
+
+  url "https://github.com/usr-sse2/RDM/releases/download/#{version}/RDM.zip"
```
Comment (vladimyr) [inline Casks/usr-sse2-rdm.rb:5]: Let's add appcast too :bulb: 
```suggestion
  url "https://github.com/usr-sse2/RDM/releases/download/#{version}/RDM.zip"
  appcast "https://github.com/usr-sse2/RDM/releases.atom"
```

File: Casks/usr-sse2-rdm.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  name "RDM"
+  desc "Set a Retina display to custom resolutions"
+  homepage "https://github.com/usr-sse2/RDM"
+
+  depends_on macos: ">= :sierra"
+
+  app "RDM.app"
+
+  uninstall quit:    "RDM",
```
Comment (vladimyr) [inline Casks/usr-sse2-rdm.rb:14]: Not that important but using the more specific form can only improve things I guess
```suggestion
  uninstall quit:    "net.alkalay.RDM",
```

**Review by vladimyr — DISMISSED** [review]

File: Casks/usr-sse2-rdm.rb (hunk 2 above)
Comment (SMillerDev) [reply Casks/usr-sse2-rdm.rb:5]: I don't think appcasts are needed when a livecheck is available?

Comment (vladimyr) [reply Casks/usr-sse2-rdm.rb:5]: Hm, am I getting out of HB(C) contributing shape again? :smile:

Comment (vladimyr) [reply Casks/usr-sse2-rdm.rb:5]: Doing cursory docs glance doesn't really reveal the answer to this question :thinking:

Comment (SMillerDev) [reply Casks/usr-sse2-rdm.rb:5]: See the `note` here: https://docs.brew.sh/Cask-Cookbook#stanza-appcast

Comment (vladimyr) [reply Casks/usr-sse2-rdm.rb:5]: I did a Github search for appcast references in markdown files, OFC that part you just referenced got collapsed inside the search result. :facepalm:

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**